src/main/java/org/mastodon/mamut/tomancak/TomancakPlugins.java

```diff
@@ -397,10 +398,14 @@ private void mergeProjects()
 
 				final Dataset dsA = new Dataset( pathA );
 				final Dataset dsB = new Dataset( pathB );
-				
+
 				final ProjectModel projectMerged = ProjectCreator.createProjectFromBdvFile( dsA.project().getDatasetXmlFile(), pluginAppModel.getContext() );
-				final MergeDatasets.OutputDataSet output = new MergeDatasets.OutputDataSet( projectMerged );
+				final MergeDatasets.OutputDataSet output = new MergeDatasets.OutputDataSet( projectMerged.getModel() );
 				MergeDatasets.merge( dsA, dsB, output, distCutoff, mahalanobisDistCutoff, ratioThreshold );
+
+				pluginAppModel.close(); // close currently open instance of Mastodon
+
+				new MainWindow( projectMerged ).setVisible( true ); // start a new instance of Mastodon that shows the result of the merge operation
```

Comment:
I would recommend to either remove the rather long comment at the end of the line (it think, it is not really needed, since it just says what is done on this line) or to put it on the line before so that one does not have to scroll to read it.

Reply:
I changed it as suggested